manifests/3.1.0/opensearch-3.1.0.yml

```diff
@@ -224,3 +224,9 @@ components:
     platforms:
       - linux
       - windows
+  - name: search-relevance
```

Comment:
Does this plugin have any dependency components? If so, we should list them out in this manifest.
e.g. https://github.com/opensearch-project/opensearch-build/blob/2565848ec951b413e821b688cc402c1bf6d4960b/manifests/3.1.0/opensearch-3.1.0.yml#L78-L86